app/views/sites/audios.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="card-audious">
+  <img src="https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/greece.jpg" />
+  <div class="card-audio-infos">
+    <div>
+    </div>
+    <h2 class=""></h2>
+  </div>
+</div>
+<%= Test %>
+<% @videos.each do |video| %>
```

Comment:
Videos in audios.html.erb ? Do you mean audios?

Reply:
Yes I mean audios, it was a mistake that I do in the beginning but later I changed it. you can see the changes in last page. Like =>
<img width="523" alt="Screenshot 2021-03-10 at 20 29 05" src="https://user-images.githubusercontent.com/74418527/110693416-4f1e2d80-81df-11eb-8564-6f94e4ef54c1.png">